<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MagnifyOptions</class>
 <widget class="QWidget" name="MagnifyOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>201</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="5" column="6">
    <widget class="QDoubleSpinBox" name="COHighDoubleSpinBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="7">
    <widget class="QLabel" name="COWavelengthValLabel">
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QSlider" name="COWavelengthSlider">
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="7">
    <widget class="QLabel" name="DoubleSliderValLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QDoubleSpinBox" name="COLowDoubleSpinBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <double>2.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="4" colspan="3">
    <widget class="QDoubleSpinBox" name="COWavelengthSpinBox">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QLabel" name="BpmSpacer">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="7" column="7">
    <widget class="QLabel" name="ChromValLabel">
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4" colspan="3">
    <widget class="QSpinBox" name="AmplificationSpinBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSlider" name="AmplificationSlider">
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="COWavelengthLabel">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Cutoff Wavelength&lt;/span&gt;&lt;/p&gt;&lt;p&gt;The higher the value, the less noise and small/short movements are amplified.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Cutoff Wavelength&lt;/span&gt;&lt;/p&gt;&lt;p&gt;The higher the value, the less noise and small/short movements are amplified.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Cutoff Wavelength:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="MagnifcationtypeComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>No Magnification</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Color Magnification</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Laplace Magnification</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Riesz Magnification</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="LevelsLabel">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Number of levels of spatial filter (image pyramid).&lt;ul&gt;&lt;li&gt;&lt;b&gt;Color Magnification:&lt;/b&gt;The higher the level, the faster the algorithm, the worse the precision of the measured signal. If level too low, the amplified signal is too low.&lt;/li&gt;&lt;li&gt;&lt;b&gt;Motion Magnification:&lt;/b&gt;The higher the level, the more big movements are amplified, so the less little movements are amplified.&lt;/li&gt;&lt;/ul&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Number of levels of spatial filter (image pyramid).&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Levels:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="LevelsSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Number of levels of spatial filter (image pyramid).&lt;/p&gt;&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Motion: The higher the value, the slower the processing, the less noise and fast/small movements&lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Color: The higher the value, the faster the processing, the more artifacts you get&lt;/li&gt;&lt;/ul&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="7" column="4" colspan="3">
    <widget class="QSpinBox" name="ChromSpinBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QLabel" name="HzSpacer">
     <property name="text">
      <string>-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="7">
    <widget class="QLabel" name="AmplificationValLabel">
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Reset Settings</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="ChromLabel">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Chrom Attenuation&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Chrominance channel magnification. The higher the value, the more colorful the movements.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Chrom Attenuation&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Chrominance channel magnification. The higher the value, the more colorful the movements.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Chrom Attenuation:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QLabel" name="loBpm">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="8">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="7">
    <widget class="QLabel" name="BpmLabel">
     <property name="text">
      <string>BPM</string>
     </property>
    </widget>
   </item>
   <item row="6" column="6">
    <widget class="QLabel" name="hiBpm">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QSlider" name="ChromSlider">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="AmplificationLabel">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Amplification factor &lt;/span&gt;&lt;/p&gt;&lt;p&gt;The higher the factor, the more movements are amplified. Reduce Amplification or enhance Cutoff Wavelength to reduce noise.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Amplification factor &lt;/span&gt;&lt;/p&gt;&lt;p&gt;The higher the factor, the more movements are amplified. Reduce Amplification or enhance Cutoff Wavelength to reduce noise.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Amplification:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="grayscaleCheckBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Converts images in grayscale before processing. Speeds up processing.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Converts images in grayscale before processing. Speeds up processing.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Grayscale</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="DoubleSliderLabel">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Frequency Range&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Upper and lower cutoff value for temporal frequency filter. Low values amplify slow movements, high values amplify fast movements.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Frequency Range&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Upper and lower cutoff value for temporal frequency filter. Low values amplify slow movements, high values amplify fast movements.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Frequency Range:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <layout class="QHBoxLayout" name="doubleSliderField"/>
   </item>
   <item row="1" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>AmplificationSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>AmplificationSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AmplificationSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>AmplificationSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ChromSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ChromSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ChromSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ChromSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
